modules/renter/downloadcache_test.go

```diff
@@ -1,36 +1,83 @@
 package renter
 
 import (
+	"container/heap"
 	"strconv"
-	"sync"
 	"testing"
 	"time"
 )
 
-// TestAddChunkToCache tests that the oldest chunk is removed
-func TestAddChunkToCache(t *testing.T) {
+// TestAdd tests that the oldest chunk is removed
+func TestAdd(t *testing.T) {
+	// Initializing minimum variables
 	udc := &unfinishedDownloadChunk{
```

Comment:
I think we don't need the whole `udc` in this test anymore after changing the `Add` method and moving the type check out of `Add`. We can just create and init the `downloadChunkCache` directly.